requirements.txt

```diff
@@ -1,8 +1,35 @@
-grpcio
-redis
-Flask
-Flask-Caching
-gunicorn
-google-api-python-client
-google-cloud-firestore
-google-cloud-pubsub
+cachetools==4.1.0
+certifi==2020.4.5.2
+chardet==3.0.4
+click==7.1.2
+Flask==1.1.2
+Flask-Caching==1.9.0
+Flask-Cors==3.0.8
+google-api-core==1.20.0
+google-api-python-client==1.9.3
+google-auth==1.17.2
+google-auth-httplib2==0.0.3
+google-cloud-core==1.3.0
+google-cloud-firestore==1.7.0
+google-cloud-pubsub==1.6.0
+googleapis-common-protos==1.52.0
+grpc-google-iam-v1==0.12.3
+grpcio==1.29.0
```

Comment:
`grpcio` needs to come top of the list due to a versioning issue with Google API.

I'll tidy this file up as there's lots in here that might be sub and/or dev dependencies.

Reply:
@kierana I've created a separate `dev-requirements.txt` to make it easier for running things locally while keeping the production environment / container clean.

Reply:
wicked, thanks